apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -805,7 +834,12 @@ export default function teacherSections(state = initialState, action) {
     }
 
     const stageExtraSettings = {};
+    const ttsAutoplayEnabledSettings = {};
     if (action.props.scriptId) {
+      // TODO: enable autoplay by default if script is a pre-reader script
```

Comment:
NIT: i'm fine with you ignoring this suggestion since it involves a slight scope increase, but imo it would be cleaner to put this behind and (D)CDO flag. that way turning it on doesn't require a deploy, and removing a flag check in a subsequent PR seems cleaner to me than uncommenting these lines. or rather, having it behind a flag seems cleaner than having it commented out.

Reply:
I'm going to avoid doing this for the scope increase (which is the reason I didn't do the work in the first place), and I would still have to open another follow-up PR to remove the flag once it's turned on.

Reply:
👍